source/NVDAObjects/IAccessible/chromium.py

```diff
@@ -185,11 +187,34 @@ class Editor(ia2Web.Editor):
 	TextInfo = EditorTextInfo
 
 
+class RedirectDocument(ia2Web.Ia2Web):
+	"""The NVDAObject for the redirect document object in Electron."""
+
+	def objectFromPointRedirect(self, x: int, y: int):
```

Comment:
Have you actually tried setting `beTransparentToMouse = True`?

Reply:
No, I haven't tried it before.

But `beTransparentToMouse = True` should only set the mouse object to the pay object for this object.

Of course I just did that and it really didn't work.